src/Soil-Core/SoilBTreePage.class.st

```diff
@@ -49,7 +49,8 @@ SoilBTreePage >> find: aKey with: aBTree [
 
 { #category : #accessing }
 SoilBTreePage >> firstItem [
-	^ items first
+
+	^ items isNotEmpty ifTrue: [ items first ]
```

Comment:
what does it return when items is empty?

Reply:
nil,maybe it is better to write is explictly in an ifFalse: